packages/sdk-redux/package.json

```diff
@@ -38,28 +33,24 @@
         "@superfluid-finance/sdk-core": "^0.1.0",
         "@reduxjs/toolkit": "^1.6.1",
         "ethers": "^5.4.5",
-        "graphql-request": "^3.5.0",
         "lodash": "^4.17.21"
     },
     "devDependencies": {
-        "@istanbuljs/nyc-config-typescript": "^1.0.1",
-        "@typescript-eslint/eslint-plugin": "^4.0.1",
-        "@typescript-eslint/parser": "^4.0.1",
-        "codecov": "^3.5.0",
-        "cspell": "^4.1.0",
-        "cz-conventional-changelog": "^3.3.0",
-        "eslint": "^7.8.0",
-        "eslint-config-prettier": "^6.11.0",
+        "@typescript-eslint/eslint-plugin": "^5.6.0",
+        "@typescript-eslint/parser": "^5.6.0",
+        "eslint": "^8.4.0",
+        "eslint-config-prettier": "^8.3.0",
         "eslint-plugin-eslint-comments": "^3.2.0",
-        "eslint-plugin-import": "^2.22.0",
-        "gh-pages": "^3.1.0",
+        "eslint-plugin-import": "^2.25.3",
         "npm-run-all": "^4.1.5",
-        "nyc": "^15.1.0",
-        "open-cli": "^6.0.1",
-        "prettier": "^2.1.1",
-        "ts-node": "^9.0.0",
+        "prettier": "^2.5.1",
+        "ts-node": "^10.4.0",
         "typedoc": "^0.22.10",
-        "typescript": "^4.3.5"
+        "typescript": "^4.5.2"
+    },
+    "peerDependencies": {
```

Comment:
I'd leave React as a mandatory peer dependency until we have a working example up with another framework.